src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -31,6 +31,9 @@ interface IPathwayMapperTableProps {
     selectedPathway: string;
     changePathway: (pathway: string) => void;
     initialSortColumn?: string;
+    columnsOverride: {
```

Comment:
This should be an optional property:
```suggestion
    columnsOverride?: {
```